pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/NamespacesTest.java

```diff
@@ -1249,9 +1249,8 @@ public void testSubscribeRate() throws Exception {
         Awaitility.await().untilAsserted(() -> assertFalse(consumer.isConnected()));
 
         // Out of limit period
-        Thread.sleep(6000L);
```

Comment:
aren't we changing the meaning of the test ?
IIUC we want to be out of the limit period.

can we verify that the consumer is disconnected and then riconnected ?

Reply:
Right, maybe we can assert is disconnected first and then wait for reconnection.

Reply:
I think it was fine to remove Thread.sleep. the delay it could be introduced with Awaitility's pollDelay

Reply:
@lhotari I have added pollDelay, PTAL.